res/qml/LibraryControl.qml

```diff
@@ -0,0 +1,188 @@
+    Mixxx.ControlProxy {
+        group: "[Playlist]"
+        key: "SelectTrackKnob"
+        onValueChanged: {
+            if (value != 0) {
+                root.focusWidget = 3;
+                root.moveSelection(value);
+            }
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Playlist]"
+        key: "SelectPrevTrack"
+        onValueChanged: {
+            if (value != 0) {
+                root.focusWidget = 3;
+                root.moveSelection(-1);
+            }
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Playlist]"
+        key: "SelectNextTrack"
+        onValueChanged: {
+            if (value != 0) {
+                root.focusWidget = 3;
+                root.moveSelection(1);
+            }
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Library]"
+        key: "MoveVertical"
+        onValueChanged: {
+            if (value != 0 && root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView)
+                root.moveSelection(value);
+
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Library]"
+        key: "MoveUp"
+        onValueChanged: {
+            if (value != 0 && root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView)
+                root.moveSelection(-1);
+
+        }
+    }
+
+    Mixxx.ControlProxy {
+        group: "[Library]"
+        key: "MoveDown"
+        onValueChanged: {
+            if (value != 0 && root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView)
+                root.moveSelection(1);
+
+        }
+    }
+
+    Mixxx.ControlProxy {
+        id: numDecksControl
+
+        group: "[Master]"
+        key: "num_decks"
+    }
+
+    Instantiator {
+        model: numDecksControl.value
+
+        delegate: LibraryLoadSelectedTrackControl {
+            group: "[Channel" + (index + 1) + "]"
+            enabled: root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView
+            onLoadTrackRequested: root.loadSelectedTrack(group, play)
+        }
+
+    }
+
+    Mixxx.ControlProxy {
+        id: numPreviewDecksControl
+
+        group: "[Master]"
+        key: "num_preview_decks"
+    }
+
+    Instantiator {
+        model: numPreviewDecksControl.value
+
+        delegate: LibraryLoadSelectedTrackControl {
+            group: "[PreviewDeck" + (index + 1) + "]"
+            enabled: root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView
+            onLoadTrackRequested: root.loadSelectedTrack(group, play)
+        }
+
+    }
+
+    Mixxx.ControlProxy {
+        id: numSamplersControl
+
+        group: "[Master]"
+        key: "num_samplers"
+    }
+
+    Instantiator {
+        model: numSamplersControl.value
+
+        delegate: LibraryLoadSelectedTrackControl {
+            group: "[Sampler" + (index + 1) + "]"
+            enabled: root.focusWidget == FocusedWidgetControl.WidgetKind.LibraryView
+            onLoadTrackRequested: root.loadSelectedTrack(group, play)
+        }
+
+    }
```

Comment:
could this also be encapsulated as an inline component?

Reply:
I considered making it a separatre component, but I think it because harder to read that way.